### PR TITLE
gov: migrate valid pingcap team to xxx-reviewers team

### PR DESCRIPTION
**Pull request by tisonkun**
@MrCroxx please notify your sponsor to approve this PR. cc @yiwu-arbug @tabokie 

@aytrack @cosven @jebter @lilinghai @shuke987 please find your sponsor to promote you as TiKV reviewers.

cc @zhangjinpeng1987 @BusyJay for the previous qa team, they are already granted [triage permission of tikv repo](https://github.com/orgs/tikv/teams/qa/repositories). And their works on triage and manage issue seem over the line to be a reviewer. What do you think?

Though I may not be the person who promote reviewers, but let's say we are still in the migration stage. Leaving a "pingcap" team is always a bad case to #118 .

I will draft the details of governance ref to #124 and then we have a formal ( and simple by major consensus ) promoting process so that we can follow the process to promote and add reviewers.

### CODE REVIEW

**Comment by tisonkun** [comment]
If you have a strong concern on the process, we can postpone this pr until process become clear.

File: teams/tikv/team.json [hunk 1]
```diff
@@ -52,15 +52,21 @@
     "reviewers": [
         "Fullstop000",
         "Jibbow",
+        "MrCroxx",
```
Comment (tabokie) [inline teams/tikv/team.json:55]: I didn't find the promotion rule for the new team architecture. Based on existing rule for engine SIG, @MrCroxx is not yet qualified for reviewer position.
```suggestion
```

Comment (tabokie) [reply teams/tikv/team.json:55]: I do find it a bit tiresome to manually give interns triage permission in certain repos where the review request isn't managed by bot. But it is still manageable for now.

**Comment by tisonkun** [comment]
https://github.com/tikv/community/pull/130#discussion_r654370421 Yep. It seems I'm in a hurry further more. Closed for now.

File: teams/tikv/team.json (hunk 1 above)
Comment (tisonkun) [reply teams/tikv/team.json:55]: Which repos is involved in this case? Why do you have to give someone triage permission while they are not reviewer?

Be aware that PingCAP doesn't take a special role in TiKV community and we are supposed to introduce concept like "A PingCAP Intern".

Comment (tabokie) [reply teams/tikv/team.json:55]: `/cc` won't work in many repos, e.g. tikv/rocksdb,tikv/rust-rocksdb,tikv/raft-engine, in which case interns without triage permission can't request review on their own.

Comment (tisonkun) [reply teams/tikv/team.json:55]: @tabokie Yep. And the moment a reviewer or committer should help in the way and the person who is not a reviewer can use `cc @reviewer` as a basic mention.

Many projects do not depend on this conventions and if reviewers active on those repo the case should be eased.

A problem is, if someone is not "A PingCAP Intern", he/she suffers the same problem, why "A PingCAP Intern" so special?

Comment (tisonkun) [reply teams/tikv/team.json:55]: Of course another way think of it is we can enable auto cc mechanism. But any is not the reason we setup a special role for PingCAP Intern in TiKV community. A good case is, an intern as a general community member, acts well and deserves his/her reviewer role.

Comment (tabokie) [reply teams/tikv/team.json:55]: Agreed. A special role would be overkill.

**Pull request closed without merging**